lib/miq_automation_engine/service_models/miq_ae_service_vm_reconfigure_request.rb

```diff
@@ -0,0 +1,9 @@
+module MiqAeMethodService
+  class MiqAeServiceVmReconfigureRequest < MiqAeServiceMiqRequest
```

Comment:
We will also need this service model for `VmCloudReconfigureRequest`.  Do want to add that here or in a separate PR?

Reply:
@gmcculloug Added VmCloudReconfigureRequest service model.